rbi/annotations/rainbow.rbi

```diff
@@ -15,7 +15,7 @@ module Rainbow
     sig { params(hex: String).returns([Integer, Integer, Integer]) }
     def self.parse_hex_color(hex); end
 
-    class Indexed < Color
+    class Indexed < Rainbow::Color
```

Comment:
Same here, the class is not flattened properly so the names are not fully qualified.